### ISSUE TITLE
Edit timesieries (mobility) description and metadata through the Data Browser

### ISSUE THREAD

**Issue by sseppi** [noi-techpark/it.bz.opendatahub.databrowser#455]
As an Open Data Hub Content Editor and Station Manager I want to edit the description and metadata of all timeseries/stations managed through the NINJA APIs.

To implement this issue we need to wait a change to the NINJA API, please consider tjh

**Comment by sseppi** [comment]
@clezag and @RudiThoeni 

To implement this issue could we use the features implemented in the issue #505?

**Comment by clezag** [comment]
@sseppi No, in #505 we implemented everything on the content side only, mobility will then just pull the data from there
